pyca/db.py

```diff
@@ -83,6 +83,7 @@ class Status(Constants):
     FAILED_UPLOADING = 6
     FINISHED_UPLOADING = 7
     PARTIAL_RECORDING = 8
+    FINISHED_RECORDING_PAUSED = 9
```

Comment:
The status `finished recording paused` is somewhat confusing for users if you don't already know what it means. How can you pause a recording if it is already finished? What about using:

```suggestion
    PAUSED_AFTER_RECORDING = 9
```

Reply:
I am fine with it, by the way that was your suggestion a while ago. ;)
But then this must also be changed i the ``` Event.vue ```